### ISSUE TITLE
All commands on "help"

### ISSUE THREAD

**Issue by jiff88** [StrongJoshua/libgdx-inGameConsole#24]
Please, can "_help_" command list contain "_help_", "_exit_", "_saveLog_" as well? I **MyCommandExecutor** i did it like:

```java
/**
 * keep it here for "help" hint
 */
@Override
public void help()
{
    super.help();
}

/**
 * keep it here for "help" hint
 */
@Override
public void exitApp()
{
    super.exitApp();
}

/**
 * keep it here for "help" hint
 */
@Override
public void printLog(String path)
{
    super.printLog(path);
}
```

but those methods are final now.
I would like to get all commands after "_help_".

Thank You!
J.

**Comment by StrongJoshua** [comment]
I have just written this, it should be uploaded as a new version soon.

**Comment by jiff88** [comment]
Thx!

**Comment by StrongJoshua** [comment]
Released in 0.5.3
